### PR TITLE
Made some improvements to the service worker

### PR DESCRIPTION
**Pull request by adnanusman**
I'm still working on loading all assets when service worker gets enabled.

### CODE REVIEW

File: game/src/js/sw.js [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
   e.waitUntil(
     caches
-      .open(STATIC_CACHE)
+      .open(RM_CACHE)
       .then(cache => {
         fetch('../../assets/assets.json').then(function(response) {
           return response.json();
         }).catch(function(err) {
           console.log('fetch:', err);
         }).then(function(files) {
```
Comment (ignacioxd) [inline game/src/js/sw.js]: Maybe `files` here should be named something more descriptive like `assets` or `assetManifest`.

File: game/src/js/sw.js [hunk 2]
```diff
@@ -1,23 +1,17 @@
 /* global Promise */
-const STATIC_CACHE = 'gosch-static-cache-v2';
-const ASSETS_CACHE = 'gosch-assets-cache-v2';
-const CACHES = [
-  STATIC_CACHE,
-  ASSETS_CACHE
-];
+const RM_CACHE = 'raginmagin-cache-v4';
```
[the remark this answers is not in the document]
Comment (adnanusman) [reply game/src/js/sw.js]: lol.. that's a typo I'll fix in next commit

File: game/src/js/sw.js (hunk 1 above)
Comment (adnanusman) [reply game/src/js/sw.js]: okay I'll change that.

**Comment by adnanusman** [comment]
Made requested changes, I'm still working on loading all of the assets at the same time so this might change a bit later but I'll add that on another PR.

**Pull request merged**